.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,6 +2,7 @@
 name: Feature forespørsel
 about: Ny feature
 title: 'Feature: kort beskrivelse av feature'
+labels: 'enhancement'
```

Comment:
feature?
```suggestion
labels: 'feature'
```

Reply:
Vi har ikke "feature"-label i repoet, men "enhancement" som er ment for features:)